### PR TITLE
fix creating unpayable stream && rename call name

### PR DESCRIPTION
**Pull request by rjman-ljm**

### CODE REVIEW

File: pallets/streaming/src/lib.rs [hunk 1]
```diff
@@ -100,17 +100,19 @@ pub mod pallet {
...
+        /// The rate per second calculated is zero
+        InvalidRatePerSecond,
         /// The stream id is not found
         InvalidStreamId,
-        /// Caller is not the streamer
-        NotTheStreamer,
-        /// Caller is not the recipient
-        NotTheRecipient,
+        /// Caller is not the stream sender
+        NotTheStreamSender,
```
Comment (GopherJ) [inline pallets/streaming/src/lib.rs:113]: can be simplified as:

- NotTheSender
- NotTheRecipient

**Review by GopherJ — APPROVED** [review]

File: pallets/streaming/src/lib.rs (hunk 1 above)
Comment (rjman-ljm) [reply pallets/streaming/src/lib.rs:113]: yep

**Comment by GopherJ** [comment]
can you rename also to:

- create_stream -> create
- withdraw_from_stream -> withdraw
- cancel_stream -> cancel

cc @i5possible

**Comment by rjman-ljm** [comment]
> can you rename also to:
> 
> * create_stream -> create
> * withdraw_from_stream -> withdraw
> * cancel_stream -> cancel
> 
> cc @i5possible
I think it would be better

**Comment by rjman-ljm** [comment]
@GopherJ  I tried it and found it would change a lot, maybe do it in the next PR.

**Pull request merged**